attestation-service/attestation-service/src/policy_engine/opa/mod.rs

```diff
@@ -146,7 +147,7 @@ impl PolicyEngine for OPA {
             bail!("OPA Policy Engine only support .rego policy");
         }
 
-        let policy_bytes = base64::engine::general_purpose::URL_SAFE_NO_PAD
+        let policy_bytes = base64::engine::general_purpose::STANDARD
```

Comment:
Note that we are changing base64 encoding here, which would cause compability problem. please ensure this is intended.

Reply:
According to feedback from some AS users, it is more reasonable to change to the standard encoding format, as URL secure padding will prohibit `\n` at the end of policy content, which is unfriendly for users to manually copy policy content and perform base64 encoding on the command line.

But this is not what this PR needs to pay attention to, so I have temporarily removed the modifications here. In the future, we can implement this modification in the new PR.